app/models/provider.rb

```diff
@@ -167,10 +167,28 @@ def self.include_courses_counts
     ).select("provider.*, COALESCE(a.courses_count, 0) AS included_courses_count")
   end
 
+  def self.include_accredited_courses_counts(provider_code)
```

Comment:
A new statement had to be created because we have need to inject the `accredited_body_code`